src/components/base/Markdown/index.js

```diff
@@ -11,6 +11,7 @@ export const Notes = styled(Box).attrs({
   fontSize: 4,
   color: 'smoke',
   flow: 4,
+  mb: 6,
```

Comment:
margins are contextual and should not leak from component.
prefer using `mb={6}` on `<Notes />` directly inside `ReleaseNotesBody` component